project.clj

```diff
@@ -1,16 +1,30 @@
-(defproject de.ubercode.clostache/clostache "1.5.0-SNAPSHOT"
+(defproject com.tradeswell/clostache "1.6.0-SNAPSHOT"
   :min-lein-version "2.0.0"
   :description "{{ mustache }} for Clojure"
   :url "http://github.com/fhd/clostache"
   :license {:name "GNU Lesser General Public License 2.1"
             :url "http://www.gnu.org/licenses/lgpl-2.1.txt"
             :distribution :repo}
-  :dependencies [[org.clojure/clojure "1.5.0"]
-                 [org.clojure/core.incubator "0.1.2"]]
-  :profiles {:dev {:dependencies [[org.clojure/data.json "0.1.2"]
-                                  [jline/jline "0.9.94"]]
+  :dependencies [[org.clojure/clojure "1.11.1"]]
+  :profiles {:dev {:dependencies [[org.clojure/data.json "2.4.0"]
+                                  [jline/jline "3.0.0.M1"]]
                    :resource-paths ["test-resources"]}
-             :1.5 {:dependencies [[org.clojure/clojure "1.5.0"]]}}
-  :repositories {"clojure-releases" "http://build.clojure.org/releases"}
-  :aliases {"all" ["with-profile" "dev:dev,1.5"]}
+             :1.11 {:dependencies [[org.clojure/clojure "1.11.1"]]}}
```

Comment:
Can we just drop this profile if it's just using the same dependencies as default?

Reply:
done